docs/content/middlewares/overview.md

```diff
@@ -11,6 +11,11 @@ There are several available middleware in Traefik, some can modify the request,
 
 Pieces of middleware can be combined in chains to fit every scenario.
 
+!!! important "Provider Namespace"
+
+    Be aware of the concept of Providers Namespace described in the [Configuration Discovery](../providers/overview.md) section. 
```

Comment:
I would add the anchor for the specific part in the provider's overview page:

```suggestion
    Be aware of the concept of Providers Namespace described in the [Configuration Discovery](../providers/overview.md#provider-namespace) section. 
```

Also what about a "warning" section for this, i think that could be less easily missed out, WDYT ?

Reply:
Changed important to warning.